src/pages/LandingPage.js

```diff
@@ -77,8 +77,8 @@ const LandingPage = () => {
 
   useEffect(() => {
     const loadProjects = withErrorHandling(async error => {
-      const errorJSON = await error.json()
-      console.log(`Unable to load billing projects due to: ${errorJSON?.message}`) // eslint-disable-line no-console
+      const errorObj = await error instanceof Response ? error.json() : error
```

Comment:
If the request fails because the session timed out, `error` will be an Error instead of a Response.